js/index.js

```diff
@@ -0,0 +1,25 @@
+var today = new Date();
+var thisYear = today.getFullYear();
+const footer = document.querySelector("footer");
+const copyright = document.createElement("p");
+
+copyright.innerHTML = `Musab ${thisYear}`;
+footer.appendChild(copyright);
+
+const skills = ["Mathematic", "Phisic", "Chemstry", "Biology", "Astronomy", "Electric"];
+
+const skillsSection = document.getElementById("skills");
+
+const skillsList = skillsSection.lastElementChild;
```

Comment:
One other thing you could have done here is use `getElementsByTagName` and then selected the first `ul` element. But what you did works as well. It just that it assumes the `ul` is always the last element